README.md

```diff
@@ -64,6 +64,13 @@ From Administrator's point of view, every Refund request results in creating two
    WKHTMLTOPDF_PATH=/usr/local/bin/wkhtmltopdf # Change this! :)
    ```
 
+1. If you are using the symfony workflows instead winzou_state_machine please import configuration:
```

Comment:
```suggestion
1. If you are using the Symfony Workflow instead of Winzou State Machine, please import configuration:
```

Reply:
But I think we can add this configuration directly to the plugin, at most it will not be used if someone is still using Winzou, wdyt?